.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true;

```

Comment:
Please add a comment linking to http://editorconfig.org/ so people know what it is